po/es.po

```diff
@@ -664,32 +673,32 @@ msgid ""
 "Try ``%s --help'' for more information.\n"
 msgstr ""
 "Argumentos no reconocidos.\n"
-"Intente ``%s --help'' para más información.\n"
+"Pruebe ``%s --help'' para obtener más información.\n"
 
 #: hardinfo/util.c:903
 #, c-format
 msgid "Module \"%s\" depends on module \"%s\", load it?"
-msgstr "El módulo \"%s\" depende del módulo \"%s\", ¿desea cargarlo?"
+msgstr "El módulo «%s» depende del módulo «%s»; ¿desea cargarlo?"
 
 #: hardinfo/util.c:926
 #, c-format
 msgid "Module \"%s\" depends on module \"%s\"."
-msgstr "El módulo \"%s\" depende del módulo \"%s\"."
+msgstr "El módulo «%s» depende del módulo «%s»."
```

Comment:
it seems maybe in multilang environment will have some unexpected results.. so for sure.. let them as original with ´\"%s\"´ and in future version next development release make a new pull request with changed to quotation

Reply:
I don't think so, it is a literal string. Can you elaborate more your answer? I've programmed in several languages and can't imagine how could it fail, I'm just curious... The only think that comes to my mind is a system with only 8 bit ASCII (not extended ASCII) support, but then it would fail anyway because accents aren't present in 8 bit ASCII and wouldn't be rendered correctly either.